python/ray/autoscaler/aws/node_provider.py

```diff
@@ -34,9 +34,12 @@ def from_aws_format(tags):
     return tags
 
 
-def make_ec2_client(region, max_retries):
+def make_ec2_client(region, max_retries, aws_credentials):
```

Comment:
Let's make sure this defaults to None?
and then below,
```
aws_credentials = aws_credentials or {}

Reply:
Done.